server/paymentProviders/stripe/virtual-cards.js

```diff
@@ -51,3 +58,173 @@ export const assignCardToCollective = async (cardNumber, expireDate, cvv, collec
 
   return await models.VirtualCard.create(cardData);
 };
+
+export const processTransaction = async (stripeEvent, stripeSignature) => {
+  const stripeTransaction = stripeEvent.data.object;
+
+  const virtualCard = await models.VirtualCard.findOne({
+    where: {
+      id: stripeTransaction.card,
```

Comment:
I don't get how this works. On our side, id is an uuid that we generate ourselves?

Reply:
On our side, id is currently set as the stripe card id or the privacy card token

Reply:
Ok!